### PR TITLE
Create cincinnati.md

### PR DESCRIPTION
**Pull request by bschonec**
Learn more about Cincy.

### CODE REVIEW

File: cincinnati.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+ # Cincinnati
+ 
+ - Famous for Cincinati style Chili
+ - AKA "Porkopolis"
+ 
+ Cincinnati-style chili is a sauce usually used over spaghetti or hot dogs, containing a unique spice blend that gives it a very distinct taste. Officially, the recipe for Skyline Chili is a well-kept family secret among Lambrinides' surviving children. However, many Skyline patrons believe that the unique taste of Skyline Chili comes from chocolate and cinnamon, spices common in Greek cuisine's meat dishes. The general recipe is not unique to Skyline—"Cincinnati-style" chili is sold by several chili parlors in the area including Empress, Dixie, Gold Star Chili, Camp Washington, Chili Time, Price Hill Chili, Dawson's School House of Chili, and other chili parlors.
```
Comment (spfortier) [inline cincinnati.md:6]: The chili sounds good.

**Comment by Pete-Ter** [comment]
Wow, impressive list

File: cincinnati.md (hunk 1 above)
Comment (bschonec) [inline cincinnati.md:6]: Skyline or Goldstar?  Please chose the former!

@spfortier

Comment (spfortier) [inline cincinnati.md:6]: I will when I get over that way.

Comment (skwgriffin) [inline cincinnati.md:6]: Know anywhere where I can get a Gold Star Chili tshirt?

Comment (bschonec) [inline cincinnati.md:6]: @skwgriffin for shame!  For shame!  Goldstar?  Them's fighting words!  ;-)

Comment (bschonec) [inline cincinnati.md:6]: @spfortier looks good

Comment (skwgriffin) [inline cincinnati.md:6]: @bschonec ... Asking for a friend!  I swear!  ;)

**Comment by bschonec** [comment]
nuking the branch!

**Pull request merged**